### ISSUE TITLE
Problem uploading funds

### ISSUE THREAD

**Issue by MGrotegut** [brave/browser-laptop#6074]
**Did you search for similar issues before submitting this one?**
Yes.

**Describe the issue you encountered:**
I opened an account with Cubits.com and sent them money.

**Expected behavior:**
Cannot upload funds to my Brave browser wallet.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Win 8.1

- Brave Version:
0.12.12

- Steps to reproduce:
    1. Click the Add funds button
    2. Do the steps described there.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by mrose17** [comment]
hi. what happens after you fund your bitcoin wallet at cubits.com ?

**Comment by MGrotegut** [comment]
I solved the issue: Using Firefox on Cubits.com it works, funds can be sent.
Before I had tried it with both IE and Brave. So it's a Cubits Problem.

But there are two new issues now: my wallet in Brave disappered - and with it the funds in it. :-( 
And there CSV file after sending monthly payments is empty.
I understand that I am working with a beta Version of Brave, but you might want to fix that all.

**Comment by mrose17** [comment]
hi. sorry to drop the thread on this. what does it mean when you say that the wallet disappeared? do you have a record at cubits with the address that you transferred funds to? if so, send me that address and the cubits address and i can refund your transfer... sorry!

**Comment by MGrotegut** [comment]
Hello mrose17:

That would be very Kind of you and highly appreciated!

My wallet in BRAVE: 3311x4j3cSFLhvVZv8XogBHvNSHnchmvYR

Amount was about 0.06944647 BTC in BRAVE...

And my Bitcoin email ist: mg@mgrotegut.de

**Comment by mrose17** [comment]
thanks. i've done the "refund" ... can you email me a copy of the top-half of about:preferences#payments -- just the grey part like this, so i can see what your browser is saying? thanks!

<img width="864" alt="0" src="https://cloud.githubusercontent.com/assets/400552/22123746/b3eaf18a-de41-11e6-9726-8c96ea007b67.png">

**Comment by MGrotegut** [comment]
Yes, sure. I just did.But here again, of course:A click on "Create Wallet" does not have any Action!"Marshall T. Rose" <notifications@github.com> hat am 19. Januar 2017 um 21:21 geschrieben:thanks. i've done the "refund" ... can you email me a copy of the top-half of about:preferences#payments -- just the grey part like this, so i can see what your browser is saying? thanks!—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by mrose17** [comment]
@MGrotegut - thanks, but for some reason, i don't see the image... could you repost it? thanks!

**Comment by mrose17** [comment]
thanks! could you go ahead and click the "Create Wallet" button and see what happens?

**Comment by MGrotegut** [comment]
Hello again, Marshall!I did that, and it happened: Nothing! No effect at all! No error message. No new wallet. NOTHING!"Marshall T. Rose" <notifications@github.com> hat am 20. Januar 2017 um 02:05 geschrieben:thanks! could you go ahead and click the "Create Wallet" button and see what happens?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by mrose17** [comment]
thanks. that's really quite odd. let's try one last experiment. find the file ledger-state.json and delete it. wait for the next browser update, install it, restart it, and see if you can create a wallet with that. sorry!

**Comment by MGrotegut** [comment]
Hi again, Marshall!Yeah, deleting that file helped, and now I could create a wallet.Thanks!"Marshall T. Rose" <notifications@github.com> hat am 23. Januar 2017 um 19:04 geschrieben:thanks. that's really quite odd. let's try one last experiment. find the file ledger-state.json and delete it. wait for the next browser update, install it, restart it, and see if you can create a wallet with that. sorry!—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by mrose17** [comment]
great, thank you for your patience!
